test/getArgumentsTest.js

```diff
@@ -0,0 +1,42 @@
+'use strict'
+
+import test from 'ava'
+
+const getArguments = require('../lib/getArguments')
+let route = '/user/:id'
+
+test('it doesn\'t spit out a name with a slash', t => {
+  let placeholder = 'iamthe/last'
+  let url = '/user/' + placeholder
+  t.notSame(getArguments(url, route), ['iamthe, last'])
```

Comment:
this is probably supposed to be an array with two elements, right now it only has 1 string